pyfr/plugins/tavg.py

```diff
@@ -357,3 +362,248 @@ def __call__(self, intg):
                     self.tstart_acc = intg.tcurr
 
                 self.tout_last = intg.tcurr
+
+
+class TavgCLIPlugin(BaseCLIPlugin):
+    name = 'tavg'
+
+    @classmethod
+    def add_cli(cls, parser):
+        cmd = cls.name
+        cp = parser.add_parser(cmd, help=f'{cmd} --help')
+        sp = cp.add_subparsers(dest='sub_cmd')
```

Comment:
Is this dest needed?

Reply:
yeah, it allows us to have the command as `pyfr tavg merge ...`

Reply:
But is the `dest` bit needed?  As we never access `sub_cmd` as far as I can tell.